test/edu-benefits/pages/directDeposit.unit.spec.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig1990n from '../../../src/js/edu-benefits/1990n/config/form';
+import formConfig1990e from '../../../src/js/edu-benefits/1990e/config/form';
+import formConfig5490 from '../../../src/js/edu-benefits/5490/config/form';
+
+const pageTests = (page, isRequired = false) => {
+  const { schema, uiSchema } = page;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input').length).to.equal(4);
+  });
+
+  if (isRequired) {
```

Comment:
I can see how this is cleaner, but it leaves room for programmer error. What if we make a change later to require these fields in another form but forget we have to require them in the test explicitly?

Can we instead check to see if `schema.properties.bankAccount.required` is not empty? Better yet, can we recursively check to see if there are any `required` arrays that aren't empty in the `schema`?